.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
I usually have my dev environment related artifacts listed in my global `~/.gitignore`. That way it doesn't leak to other projects.

```gitignore
# ~/.gitignore
.idea/*
```

Reply:
sigh I knew that was gonna happen